test/pom.xml

```diff
@@ -189,6 +190,30 @@ THE SOFTWARE.
       <version>318.va_f3ccb_729b_71</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
```

Comment:
[`junit-jupiter` is an aggregator](https://junit.org/junit5/docs/current/user-guide/#dependency-metadata-junit-jupiter) which pulls `junit-jupiter-api` and `junit-jupiter-engine` listed below.

Reply:
Why are you adding them separately when using the aggregator?

Reply:
To be consistent with `cli/pom.xml`, which is code I did not write.

Reply:
https://github.com/jenkinsci/jenkins/pull/6842

Reply:
@daniel-beck In the future, I am asking that you refrain from commenting on my PRs with (implicit) requests for change regarding pre-existing issues.

Reply:
It wasn't. I misread offa's comment as yours and providing an explanation, rather than as reviewer's feedback. That explanation made no sense to me, hence the comment.